code/src/components/ThoughtsForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const ThoughtsForm = ({ newThought, onNewThoughtChange, onFormSubmit }) => {
+  return (
+    <div className="form-container">
+      <form onSubmit={onFormSubmit}>
+        <h1>What is making you happy right now?</h1>
+        <textarea
+          value={newThought}
+          onChange={onNewThoughtChange}
+          placeholder="Type your happy thoughts here..."
+          maxLength="140" />
+        <div className="counter">{0 + newThought.length} / {140 - newThought.length} </div>
+        <button
+          className="submit-btn"
+          type="submit">
+          <span className="emoji" role="img" aria-label="heart-emoji">❤️</span> Send happy thought! <span role="img" aria-label="heart-emoji">❤️</span>
```

Comment:
I do not understand why you put emoji in <span>?

Reply:
The emoji here is only as a nice detail on the button.